src/program/deploys.js

```diff
@@ -73,7 +73,10 @@ module.exports.deploy = (api, config) => (stage, reference, options) => {
       spinners.succeedAndNext()
       return { projectId, token }
     }))
-    .catch(() => spinners.fail())
+    .catch(() => {
+      spinners.fail()
+      h.fail(`Error: getAuthenticityToken(${projectId}, ${stage})`)
```

Comment:
I think you should catpure the exception and display the error message.
+ should we continue the execution ? Here the catch doesn't throw the error, so it continue the deploiement. Is it the wanted behavior ?